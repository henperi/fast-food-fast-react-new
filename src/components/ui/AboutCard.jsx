import React from 'react';

const AboutCard = () => (
  <div className="flex-item card about" id="about">
    <h2>About Fast Food Fast</h2>
    <p className="text-simple">
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem
      ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum
      dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
    </p>
    <p className="text-simple">
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem
      ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum
      dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
    </p>
    <p className="text-simple">
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem
      ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum
      dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
    </p>
  </div>
);

export default AboutCard;
