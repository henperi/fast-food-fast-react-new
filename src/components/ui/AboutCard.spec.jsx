import React from 'react';
import { shallow } from 'enzyme';
import AboutCard from './AboutCard';

test('should render AboutCard correctly', () => {
  const wrapper = shallow(<AboutCard />);
  expect(wrapper).toMatchSnapshot();
});
