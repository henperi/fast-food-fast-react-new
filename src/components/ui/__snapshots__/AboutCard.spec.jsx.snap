// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AboutCard correctly 1`] = `
<div
  className="flex-item card about"
  id="about"
>
  <h2>
    About Fast Food Fast
  </h2>
  <p
    className="text-simple"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora! Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
  </p>
  <p
    className="text-simple"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora! Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
  </p>
  <p
    className="text-simple"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora! Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam in sequi incidunt tempora!
  </p>
</div>
`;
